templates/workspace_services/guacamole/template_schema.json

```diff
@@ -71,6 +71,12 @@
       "title": "Expose externally",
       "description": "Is the Guacamole service exposed outside of the vnet",
       "default": true
+    },
+    "workspace_subscription_id": {
```

Comment:
Know we discussed this, not sure it's needed as

https://github.com/microsoft/AzureTRE/blob/839548ccc7ed0d445d0324fa87f602aa45f6ebfe/devops/scripts/register_bundle_with_api.sh#L102-L103

uses porter explain t get all parameters and merges with template_schema.json. Worth a test.